standard/expressions.md

```diff
@@ -245,7 +245,7 @@ An operation of the form `«op» x` or `x «op»`, where «op» is an overloada
 An operation of the form `x «op» y`, where «op» is an overloadable binary operator, `x` is an expression of type `X`, and `y` is an expression of type `Y`, is processed as follows:
 
 - The set of candidate user-defined operators provided by `X` and `Y` for the operation `operator «op»(x, y)` is determined. The set consists of the union of the candidate operators provided by `X` and the candidate operators provided by `Y`, each determined using the rules of [§12.4.6](expressions.md#1246-candidate-user-defined-operators). For the combined set, candidates are merged as follows:
-  - If `X` and `Y` are the same type, or if `X` and `Y` are derived from a common base type, then shared candidate operators only occur in the combined set once.
+  - If `X` and `Y` are the same type or runtime interchangeable, or if `X` and `Y` are derived from a common base type, then shared candidate operators only occur in the combined set once.
```

Comment:
"Common base type" implies "same" but I suspect should be runtime interchangeable. Not sure, but worth checking!

Reply:
Well, no. "common base type" could be:

```csharp
public class B
{
    public static bool operator > (B left, B right)
    {
        // ... logic
    }
}

public class D1 : B
{
}

public class D2 : B
{
}
```

The types `D1` and `D2` are derived from a common base type. The operator is only added once.